go/vt/vtgate/planbuilder/testdata/postprocess_cases.json

```diff
@@ -3303,13 +3303,13 @@
               {
                 "OperatorType": "Aggregate",
                 "Variant": "Ordered",
-                "Aggregates": "min(1) AS min(a.id)",
+                "Aggregates": "min(1|3) AS min(a.id)",
```

Comment:
is aggregation engine able to use weight_string for min/max?

Reply:
the planner now produces the correct plan, and the engine side will fail if it actually needs the `weightstring`. this way, once we fix then engine, we don't need to change anything on the planner